R/makeRandomResiduals.R

```diff
@@ -7,20 +7,22 @@
 #' @param eList named list with at least the Sample dataframe
 #' @keywords water-quality statistics
 #' @importFrom truncnorm rtruncnorm
+#' @examples 
+#' choptankAugmented <- makeAugmentedSample(Choptank_eList)
 #' @export
 #' @return eList named list with modified Sample data frame.
 makeAugmentedSample <- function(eList){
-
+  
   if(all(c("SE","yHat") %in% names(eList$Sample))){
     localSample <- eList$Sample
-    numSamples<-length(localSample$Uncen)
+    numSamples <- length(localSample$Uncen)
     a <- ifelse(localSample$Uncen==0&!is.na(localSample$ConcLow),log(localSample$ConcLow)-localSample$yHat,-Inf)
     b <- ifelse(localSample$Uncen==1,+Inf,log(localSample$ConcHigh) - localSample$yHat)
     mean <- ifelse(localSample$Uncen==1,log(localSample$ConcHigh) - localSample$yHat,0)
     sd <- ifelse(localSample$Uncen==1,0,localSample$SE)
-    localSample$rResid<-truncnorm::rtruncnorm(numSamples,a,b,mean,sd)
+    localSample$rResid <- rtruncnorm(numSamples,a,b,mean,sd)
```

Comment:
i think the going advice is to use `::` more rather than less. see 1st paragraph of 'R Functions' at http://r-pkgs.had.co.nz/namespace.html#imports